src/components/media-player/extensions/timeline/timeline.component.html

```diff
@@ -5,11 +5,13 @@
 
     <div class="buffered-bar" *ngFor="let buffer of buffered" [style.left.%]="buffer.start" [style.width.%]="buffer.end - buffer.start"></div>
 
-    <div class="media-progress-bar" [style.width.%]="position"></div>
+    <div class="media-progress-bar" [style.width.%]="position">
+        <div #progressThumb class="media-progress-bar-thumb" (mouseenter)="scrub.visible = false; pop.hide(); $event.stopPropagation()"
+            (mouseleave)="scrub.visible = true; pop.show(); $event.stopPropagation()"></div>
+    </div>
 
-    <div class="scrub-handle" [style.left.px]="scrub.position" [tooltip]="popTemplate" placement="top" triggers="" #pop="bs-tooltip"
+    <div class="scrub-handle" [class.scrub-hide]="scrub.visible === false" [style.left.px]="scrub.position" [tooltip]="popTemplate" placement="top" triggers="" #pop="bs-tooltip"
```

Comment:
Change the class name to reference the handle as scrub describes the action, not the element. Can also simplify expression eg:

```html
[class.scrub-handle-hidden]="!scrub.visible"
```